geoprocessing/src/modules/planning-units/planning-units.controller.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  Controller,
+  Logger,
+  Get,
+  Req,
+  Param,
+  UploadedFile,
+  Post,
+  UseInterceptors,
+} from '@nestjs/common';
+import { FileInterceptor } from '@nestjs/platform-express';
+
+import { apiGlobalPrefixes } from 'src/api.config';
+import { uploadOptions } from 'src/utils/file.utils';
+import { ShapeFileService } from '../shapefiles/shapefiles.service';
+
+@Controller(`${apiGlobalPrefixes.v1}/planning-units`)
+export class PlanningUnitsController {
+  private readonly logger: Logger = new Logger(PlanningUnitsController.name);
+  constructor(public shapefileService: ShapeFileService) {}
+  @UseInterceptors(FileInterceptor('file', uploadOptions))
+  @Post('/:id/planning-unit-shapefile')
+  async getShapeFile(
```

Comment:
Would be great to provide OpenAPI definitions for swagger for both input and output.

Reply:
Changes added in [here](https://github.com/Vizzuality/marxan-cloud/pull/161/commits/11a63a68962ecb3e0793d6fc77c7e223f91b1b94)